pikaday.js

```diff
@@ -1082,13 +1102,22 @@
             }
 
             for (var c = 0; c < opts.numberOfMonths; c++) {
-                randId = 'pika-title-' +
-                  Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 2);
+                randId = 'pika-title-' + c;
```

Comment:
hmm 2 questions on this.

1. why did we remove this `Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 2);` is this intentional
2. this here `randId` suppose to mean `randomId`. so by removing the random generator from this variable, now the name `randId` doesn't actually go with the logic. 🤔

Reply:
Addressed your questions:

1. I removed it on purpose
2. the `randId` to help make the title wrapper unique in case showing multi-months calendar (e.g: use case to show 2 continuous months). If we maintain the `randId`, we will have different `pika-title-**` for everytime calling this method so we cannot access the child DOM if we want to.